### PR TITLE
Bittrex - changed type of field "Id" in DepositHistory struct

### PR DESCRIPTION
**Pull request by vazha**
# PR Description

Fixes # (issue)
Fixed error in my previous PR 
## Type of change
Id field of DepositHistory type changed to int64

### CODE REVIEW

File: exchanges/bittrex/bittrex_types.go [hunk 1]
```diff
@@ -209,3 +209,18 @@ type WithdrawalHistory struct {
 		InvalidAddress bool    `json:"InvalidAddress"`
 	} `json:"result"`
 }
+
+// DepositHistory holds the Deposit history data
+type DepositHistory struct {
+	Success bool   `json:"success"`
+	Message string `json:"message"`
+	Result  []struct {
+		Id            int64   `json:"Id"`
```
Comment (shazbert) [inline exchanges/bittrex/bittrex_types.go]: change id -> ID

Comment (vazha) [reply exchanges/bittrex/bittrex_types.go]: I overlooked that in the previous PR commit (4b9de0fdd02ca55d4e97df8337b2439bf72805ae)  "Id"  was a changed ti "ID". So I can make changes in next PR or author can make correction on current PR?

Comment (shazbert) [reply exchanges/bittrex/bittrex_types.go]: Awesome, just needs a rebase or merge and we are good to go. Thanks heaps for the PR!

Comment (vazha) [reply exchanges/bittrex/bittrex_types.go]: And you know ... I’m now studying the Bitrix API documentation, and I have the feeling that their engineer is well versed in cannabis :)

Comment (thrasher-) [reply exchanges/bittrex/bittrex_types.go]: @vazha seems as though a lot of the exchange API documentation authors are high flyers 😆

**Comment by thrasher-** [comment]
There still exists a merge conflict. Once that has been corrected, then we can merge in the changes

**Comment by vazha** [comment]
> There still exists a merge conflict. Once that has been corrected, then we can merge in the changes

Dear thrasher, as I wrote earlier, this was my own commit (ID has been set as string type), and it was mistake. In current commit I changed it to int64. This is not a conflict.

**Comment by thrasher-** [comment]
Thanks @vazha! Merge commit fixed the conflict and this is good to go!

**Pull request merged**